juniper_codegen/src/common/deprecation.rs

```diff
@@ -105,9 +105,9 @@ impl ToTokens for Directive {
         let reason = self
             .reason
             .as_ref()
-            .map_or_else(|| quote! { None }, |text| quote! { Some(#text) });
+            .map_or_else(|| quote! { ::std::option::Option::None }, |text| quote! { ::std::option::Option::Some(#text) });
         quote! {
-            .deprecated(::std::option::Option::#reason)
+            .deprecated(#reason)
```

Comment:
This worked before but I changed it to make it more clear that the types are correctly qualified. I spent quite a while trying to track down an `expected identifier, found ::` error caused by the doubly qualified path (`::std::option::Option::std::option::Option::Some(...)` after only changing the other line.

Happy to revert if preferred.